tests/functional/pages/surveys/other_option/other-options.page.js

```diff
@@ -0,0 +1,26 @@
+import QuestionPage from '../question.page'
+
+class OtherOptionsPage extends QuestionPage{
```

Comment:
Looks like we are diverging from the 'QuestionPage' per page model. If that is the case we should probably rename QuestionPage to Questionnaire or something and move all the other pages into one class. I think it's easier if we stick to one pattern?